docs/environment.yml

```diff
@@ -3,19 +3,29 @@ channels:
     - conda-forge
     - defaults
 dependencies:
-    - python=3.6
-    - mock  # needed for local python 2.7 builds
-    - numpy=1.14.2
-    - scipy
-    - pytables
-    - pandas=0.22.0
-    - pytz
+    - coveralls
```

Comment:
Why do you want to change the docs environment? This has been a pain to test and debug on RTD in the past. I suggest addressing separately if there's a need.

Reply:
I wasn't sure what to do here, but I'm fine reverting.

Reply:
Ok, please do revert this change.